upload/admin/view/template/extension/module/smaily_for_opencart.tpl

```diff
@@ -327,6 +325,85 @@
                 </div>
               </div>
             </div>
+            <!-- RSS -->
+            <div id="section4" class="tab-pane fade in">
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-category"><?php echo $rss_category_title ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_category" id="rss-category" class="form-control smaily-rss-options">
+                    <option value=''>All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = is_array($rss_category) && in_array($category['category_id'], $rss_category) ? 'selected' : '';
+                      echo("<option value=$category[category_id] $selected>$category[name]</option>");
+                    }
```

Comment:
Use pattern described here https://github.com/sendsmaily/smaily-opencart-module/pull/94#discussion_r482091656 to rework this code block.

Reply:
Not echoing out HTML